css/_welcome_page.scss

```diff
@@ -4,90 +4,70 @@ body.welcome-page {
 }
 
 .welcome {
+    background-color: $welcomePageHeaderBackground;
+    display: flex;
+    flex-direction: column;
     font-family: $welcomePageFontFamily;
-    height: 100%;
+    justify-content: space-between;
+    min-height: 100vh;
     position: relative;
 
     .header {
         align-items: center;
-        background: $welcomePageHeaderBackground;
         display: flex;
         flex-direction: column;
+        min-height: fit-content;
         overflow: hidden;
         position: relative;
         text-align: center;
 
         .header-text {
             display: flex;
             flex-direction: column;
-            justify-content: space-around;
-            margin-top: 120px;
-            margin-bottom: 20px;
+            margin-top: 156px;
```

Comment:
Wouldn't it be better if this margin-top is a variable depending on the watermark header height + some additional space?